src/sentry/event_manager.py

```diff
@@ -2344,20 +2358,23 @@ def _get_severity_score(event: Event) -> tuple[float, str]:
                 extra=logger_data,
             )
             reason = "microservice_max_retry"
+            cache.set(SEER_ERROR_COUNT_KEY, cache.get(SEER_ERROR_COUNT_KEY, 0) + 1, timeout=60 * 60)
```

Comment:
It probably makes sense to move this code into a function since it's duplicated, and we might want to change the way we increment this cache